package.json

```diff
@@ -40,6 +40,7 @@
     "grunt": "1.0.1",
     "grunt-cli": "1.2.0",
     "grunt-concurrent": "2.3.1",
+    "grunt-contrib-connect": "^1.0.2",
```

Comment:
conventionally, we lock the package versions. Can you remove the `^` here, please?